tests/queries/0_stateless/01059_storage_file_brotli.sql

```diff
@@ -31,10 +31,19 @@ SELECT count(), max(x) FROM file;
 
 DROP TABLE file;
 
-SELECT count(), max(x) FROM file('data{1,2,3,4}.tsv.{gz,br,xz,zst}', TSV, 'x UInt64');
+CREATE TABLE file (x UInt64) ENGINE = File(TSV, 'data5.tsv.lz4');
+TRUNCATE TABLE file;
+
+INSERT INTO file SELECT * FROM numbers(1000000);
```

Comment:
This comment was supposed to belong to this PR but I guess I ended up commenting elsewhere:

@evillique I've noticed this behavior (before as well while I was doing the initial implementation) where inserting into file / table with large number of rows causes the query to hang. So I added this test (similar to other compression methods) to confirm and it still hangs according to my local test.

Can also reproduce using (query never completes):

tower :) insert into function file('test2.lz4', 'TSV', 'number UInt64', 'lz4') select * from numbers(0,1000000);

INSERT INTO FUNCTION file('test2.lz4', 'TSV', 'number UInt64', 'lz4') SELECT *
FROM numbers(0, 1000000)

Query id: 3fa90fbe-9bab-4d92-9aaf-79763ddc5de1

→ Progress: 1.00 million rows, 8.00 MB (9.12 million rows/s., 72.99 MB/s.)  99%

But after your changes in f2852d1a088ee55fcbf6bd9c17dfb742edf0ff8a, it now works fine.